R/load.r

```diff
@@ -94,11 +94,14 @@ load_all <- function(pkg = ".", reset = TRUE, recompile = FALSE,
   # in the DESCRIPTION file
   pkg$collate <- as.package(pkg$path)$collate
 
+  # Forcing all of the promises for the loaded namespace now will avoid lazy-load
+  # errors when the new package is installed overtop the old one.
```

Comment:
You are correct, this comment was partially copied from `install()` and I missed that. Fixed with https://github.com/jimhester/devtools/commit/1ca9e288079fef5dabcc804d747ba6c2c12a8b84 Thanks!